test/mpi_tests/bcreader_mpi_tests.jl

```diff
@@ -18,7 +18,7 @@ include(joinpath(@__DIR__, "..", "..", "artifacts", "artifact_funcs.jl"))
 const sst_data = joinpath(sst_dataset_path(), "sst.nc")
 
 # set up MPI communications context
-const comms_ctx = ClimaComms.SingletonCommsContext(ClimaComms.CPUDevice())
+const comms_ctx = ClimaComms.SingletonCommsContext(ClimaComms.CPUSingleThreaded())
```

Comment:
```suggestion
const comms_ctx = ClimaComms.SingletonCommsContext(ClimaComms.device())
```
This way this file will work with unthreaded, threaded, or gpu